src/styles/saas-list.less

```diff
@@ -14,21 +14,37 @@
   .card-view-pf .card-content {
     background-color: @color-pf-white;
     cursor: pointer;
+    margin: 0;
+
+    @media (min-width: @screen-sm-min) {
+      margin: 2px 0 10px 0;
+    }
   }
 
   .card-view-pf .card {
-    height: 200px;
-    width: 260px;
+    height: 100%;
+    width: 100%;
```

Comment:
Do the patternfly defaults for height/width/margin not work well? It seems odd that we have to set this.